src/s2e/Plugins/Lua/Lua.h

```diff
@@ -106,7 +106,7 @@ template <typename T> class Lunar {
         int status = lua_pcall(L, 1 + nargs, nresults, errfunc); // call method
         if (status) {
             const char *msg = lua_tostring(L, -1);
-            if (msg == NULL)
+            if (msg == nullptr)
```

Comment:
Use {} around ifs. Does the clang formatter catch this?

Reply:
No :(